src/modules/book_saver.rb

```diff
@@ -0,0 +1,21 @@
+require 'json'
+
+module DataSaver
+  def save_books
+    books_data = []
+
+    @books.each do |book|
+      books_data.push({ publish_date: book.publish_date, publisher: book.publisher, cover_state: book.cover_state,
+                        label: { title: book.label.title, color: book.label.color } })
+    end
+    File.write('data/books.json', JSON.pretty_generate(books_data))
+  end
+
+  def save_labels
+    labels_data = []
+    @labels.each do |label|
+      labels_data.push({ title: label.title, colour: label.color })
+    end
```

Comment:
kindly Check the the `File.write` for `label.json` file work correctly 
- i have noticed that you are saving the color `lable` in `json` file as `colour` and when you read the file `lable["color"]`

Reply:
Thanks for noticing this, I misspelt `color` as `colour`. I will make the change and push again.